src/services/error-manager.js

```diff
@@ -0,0 +1,15 @@
+const ApiErrorDeserializer = require('../deserializers/api-error');
+
+function handleError(rawError) {
+  let error;
+  // NOTICE: We check if the errors are from the API or if they've been thrown internally.
+  if (!rawError.status) {
+    error = rawError;
+  } else {
+    error = ApiErrorDeserializer.deserialize(rawError);
```

Comment:
[Error] I tried it without the backend PR and the 404 returned by the backend sends HTML (this is because of the un-existing route)

Thus we'll probably need to keep a try catch here to avoid badly formatted backend issues.

```
✖ Analyzing your setup
> SyntaxError: Unexpected token < in JSON at position 0
```